docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -421,14 +420,14 @@ helm install $HELM_RELEASE_NAME camunda/camunda-platform \
 
 #### Verification
 
-The following command will show the deployed pods of the newly created region.
+The following command will show the pods deployed in the newly created region.
```

Comment:
Should we move the actual command beneath this? It becomes less clear that the command a few lines down is related.

Reply:
Done